### ISSUE TITLE
ARIA attributes for a stack view component

### ISSUE THREAD

**Issue by dkanev** [vmware/clarity#2429]
**Select one ...** (check one with "x")

```
[X ] feature request
[X] enhancement
```

I would like to request ARIA attributes for the stack view component
Probably they should be something like the on specified by the “WAI-ARIA Authoring Practices 1.1” for Accordion (https://www.w3.org/TR/wai-aria-practices/#accordion)
For example:
The stack view is good to have ARIA attributes when the stack block is collapsed and expanded. 

Also I would like if possible to be provided ARIA roles for this component and its parts.

We have a lot of Angular JS components which use the downgrade stack block. We would like if possible the above requested improvements to be made in such a way that to be available in the downgraded stack blocks.

### Environment details

* **Angular version:** 4.x.x

* **Clarity version:** 11.0.18

* **OS and version:**

* **Browser:** all

**Comment by adityarb88** [comment]
@dkanev: We have a general issue open to add ARIA attributes to all our components here: https://github.com/vmware/clarity/issues/1190 Please subscribe to https://github.com/vmware/clarity/issues/1190 for more updates. 

@smathis: Can we close this issue as a duplicate of https://github.com/vmware/clarity/issues/1190 or do we want to track this separately?

**Comment by mathisscott** [comment]
@adityarb88 
No, we are keeping these separate so we can prioritize them according to a need of a VMware product. Let's leave them atm. Thanks.

**Comment by dkanev** [comment]
@adityarb88, @mathisscott 
Is it possible this to be merge in 0.11.x version of clarity?
Thanks!

**Comment by adityarb88** [comment]
Yes. This change will be merged in both 0.11 and 0.12
